index.html

```diff
@@ -0,0 +1,349 @@
+            <button type="button" class="btn about-me-btn">Get my resume </button>
+          </div>
+        </div>
+        <ul class="skill-list">
+            <!----<div class="frame-12"> --->
+            <li class="frame-7-languages">
+              <a href="#">Languages</a>
+              <i class='fa fa-chevron-down'></i>
+
+               <!---<ul class="language-list"> ---> 
+              <li>
+                <ul class="language-list">               
+                  <li class="frame-9-10-11">                           
+                      <img src="./images/Ellipse-1.png" alt="Ellipse 1(Javascript)">
+                      <a href="#">Javascript</a>                
+                  </li>   
+                  <li class="frame-9-10-11">                    
+                      <img src="./images/ellipse-2.png" alt="Ellipse 1(HTML)">
+                      <a href="#">HTML</a>                             
+                  </li>                         
+                  <li class="frame-9-10-11">               
+                      <img src="./images/ellipse-3.png" alt="Ellipse 3(CSS)">
+                      <a href="#">CSS</a>                    
+                  </li>
+                </ul>
+              </li>
+            
+            <li class="skill-and-framework">
+              <a href="#">Frameworks</a>
+              <i class='fa fa-chevron-right'></i>
+            </li>
+            <li class="skill-and-framework">
+              <a href="#">Skills</a>
+              <i class='fa fa-chevron-right'></i> 
+            </li>
+        </ul>
+      </div>           
+
+    </section>
+    <!---------End of about myself section-------->
+    <!---------End of mobile-finish-version----->
+
+
+    <!--------Form container mobile version------>
+    <footer>
+      <div class="form-container">
+        <img src="/images/contact-form-shapes-mobile.png" alt="Contact Form Shapes" class="contact-form-shapes">
+        <span class="contact-me-title">Contact me</span>
+        <p class="contact-me-text">
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+
+         <form action="https://formspree.io/f/mbjblbnd" method="POST">
+          <div class="input-field name">
+            <input class="form1-input name" type="text" placeholder="Enter your name please" maxlength="30" required>
+          </div>
+          <div class="input-field email">
+            <input class="form1-input" type="email" placeholder="Enter your email please" required>
+          </div>
+          <div class="input-field message" >
+            <textarea  class="form1-input" name="message" id="contact-message"  maxlength="500" placeholder="Write your message here here" required></textarea>
+          </div>
+
+          <div class="get-touch">
+            <button type="button"> Get to touch</button>
```

Comment:
- [ ] As the previous code reviewer mentioned that the `Get to touch` button should have a `type="submit"` instead of `type="button"`.

Reply:
The code reviewer reviewed it before I fixed the linters error.

Reply:
You have to make the required changes that was asked by the code reviewer